src/msg/msg_types.h

```diff
@@ -26,6 +26,9 @@
 
 #define MAX_PORT_NUMBER 65535
 
+// ceph_sockaddr_storage matches the Linux format.
+#define AF_INET6_LINUX 10
+
```

Comment:
Should we only define this in the _WIN32 case as well? Other OS's presumably won't need it.

Reply:
Makes sense.